woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1123,19 +1123,19 @@ def test_get_invalid_schema_message(sample_df):
     schema_df.ww.init(name='test_schema', index='id', logical_types={'id': 'Double', 'full_name': 'PersonFullName'})
     schema = schema_df.ww.schema
 
-    assert _get_invalid_schema_message(schema_df, schema) is None
-    assert (_get_invalid_schema_message(sample_df, schema) ==
+    assert get_invalid_schema_message(schema_df, schema) is None
```

Comment:
Can we move all of these `get_invalid_schema_message` tests to `test_accessor_utils`?

Reply:
Oh yeah. I meant to do that and obviously forgot. Thanks for catching!